src/ScheduleService.Data/TaskRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿using LT.DigitalOffice.ScheduleService.Data.Interfaces;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Data
+{
+  public class TaskRepository : ITaskRepository
+  {
+    public Task<Guid?> CreateAsync()
+    {
+      throw new NotImplementedException();
+    }
+
+    public Task FindAsync()
+    {
+      throw new NotImplementedException();
+    }
+
+    public Task GetAsync(Guid id)
+    {
+      throw new NotImplementedException();
+    }
+
+    public Task<bool> EditAsync(Guid id)
```

Comment:
pass JsonPatchDocument/EditRequest

Reply:
Should there be 2 methods in the repository: EditAsync for JsonPatchDocument and UpdateAsync for EditRequest?